packages/dev-console-app/src/ExtensionRow/ExtensionRow.tsx

```diff
@@ -32,13 +32,30 @@ export function ExtensionRow({
     development: {hidden, status},
   } = extension;
 
-  const handleSelect = useCallback(
+  const handleRowSelect = useCallback(
     (event?: MouseEvent) => {
-      if (event) event.stopPropagation();
-      onSelect(extension);
+      if (event) {
+        event.stopPropagation();
+      }
+
+      // Ignore the case of the checkbox itself being selected, which is handled by handleCheckBoxSelect
+      const target = event?.target as Element;
```

Comment:
I see, this is a limitation with the Polaris component. I think we should do this workaround then:
```tsx
<td onClick={ (event) => event.stopPropagation()}>
  <Checkbox label="" checked={selected} onChange={handleCheckboxSelect} />
</td>
```

Reply:
@vividviolet, I played with this work-around previously. I opted for the solution in this PR because clicking in the `<td>` near the checkbox doesn't toggle it. This seems like a worse user experience to me. Many users may try to toggle the actual checkbox and miss which will not activate the checkbox (see video). The existing solution in this PR doesn't have this problem. It also forces me to add 2 Prettier ignore lines to the `Extension.tsx` file.

https://user-images.githubusercontent.com/4490738/173096963-fa4602d2-3bc4-4e92-bef2-12e6b3b72477.mp4